src/handlers/comment/handlers/table.ts

```diff
@@ -16,24 +16,23 @@ export const tableComment = ({
   days?: number;
 }) => {
   return `
-<code>
```

Comment:
This was intentional to display the table in monospace.

Reply:
https://github.com/Keyrxng/UbiquityBotTesting/issues/43#issuecomment-1712886879

Reverted it back and managed to remove some of the whitespace